packages/rum-core/test/common/config-service.spec.js

```diff
@@ -209,6 +213,18 @@ describe('ConfigService', function () {
     ])
   })
 
+  it('should check unknown config options', () => {
+    const errors1 = configService.validate({
```

Comment:
nit:

What about `errors` instead?

Reply:
works for me 👍 (you can see I copied and pasted :D)